src/runtime/components/BlogDetail.vue

```diff
@@ -53,166 +54,66 @@
     <!-- CTA -->
     <div class="cb-cta-section">
       <CTA1 v-if="CTACompName == 'CTA1'" />
-      <CTA2 v-if="CTACompName == 'CTA2'" :recaptcha-key="recaptchaKey" />
-      <CTA3 v-if="CTACompName == 'CTA3'" :recaptcha-key="recaptchaKey" />
+      <CTA2
+        v-if="CTACompName == 'CTA2'"
+        :recaptcha-key="recaptchaKey"
+        :contactApiUrl="contactApiUrl"
+      />
+      <CTA3
+        v-if="CTACompName == 'CTA3'"
+        :recaptcha-key="recaptchaKey"
```

Comment:
We must use unique naming conventions everywhere. Whether its in `camel case` or `dashed` format. Whatever seems best practice